smart_campus/app/templates/app/station_new.html

```diff
@@ -1,138 +1,123 @@
 {% extends "app/base.html" %}
 {% load static %}
 
-{% block title %}Smart Campus{% endblock %}
-
 {% block stylesheet %}
-  {{ block.super }}
-  <link rel="stylesheet" type="text/css" href="{% static "app/css/station_new.css" %}" />
+  {{ block.super }}>
+  <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/trix/0.11.0/trix.css">
+  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/trix/0.11.0/trix.js"></script>
+  <link rel="stylesheet" type="text/css" href="{% static "app/css/edit.css" %}">
 {% endblock %}
 
 {% block content %}
-<div class="container content">
-  <!-- Display Error -->
-  {% for field in form %} 
-    {% for error in field.errors %}
-        <li class='text-danger'><strong>{{ field.label }}: {{ error|escape }}</strong></li>
-    {% endfor %}    
-  {% endfor %}
-  <!-- Form -->
-  <form action="" method="POST" enctype="multipart/form-data">            
-    {% csrf_token %}
-    <div class="row">
+  <div class="ui container">
+    <form class="ui centered big form" method="POST" enctype="multipart/form-data" id="station_form">
+        {% for field in form %}
+          {% if field.errors %}
+            <div class="ui negative message">
+            {% for error in field.errors %}
```

Comment:
You should unify your indentation style in django template `for` and `if` block.

Reply:
okay, fixed it.